src/main/scala/com/ing/wbaa/gargoyle/proxy/api/ProxyService.scala

```diff
@@ -31,6 +33,10 @@ trait ProxyService extends LazyLogging {
   // Authorization methods
   protected[this] def isUserAuthorizedForRequest(request: S3Request, user: User): Boolean
 
+  // Atlas Lineage
+  protected[this] def atlasSettings: GargoyleAtlasSettings
+  protected[this] def createLineageFromRequest(httpRequest: HttpRequest, userSTS: User): Future[LineagePostGuidResponse]
```

Comment:
Why are we actually returning LineagePostGuidResponse here? It's not being used.

Reply:
Same here, experimental feature